tests/unit/test_login.py

```diff
@@ -25,40 +25,44 @@ class LoginTestClass(asynctest.TestCase):
 
     async def test_handle_login(self):
         """Test initial login handler."""
-        mock_req = types.SimpleNamespace(**{
-            "query": {"navto": "http://example"}
-        })
+        mock_req = types.SimpleNamespace(**{"query": {"navto": "http://example"}})
         resp = await swift_browser_ui.login.handle_login(mock_req)
 
-        self.assertEqual(resp.headers['Location'], "/login/front")
+        self.assertEqual(resp.headers["Location"], "/login/front")
         self.assertEqual(resp.status, 302)
 
     async def test_sso_query_begin_with_trust(self):
         """Test sso query begin function."""
-        with unittest.mock.patch("swift_browser_ui.login.setd", new={
+        with unittest.mock.patch(
+            "swift_browser_ui.login.setd",
+            new={
                 "auth_endpoint_url": "https://example.os.com:5001/v3",
                 "set_origin_address": "https://localhost/login/websso",
                 "has_trust": True,
-        }):
+            },
+        ):
             resp = await swift_browser_ui.login.sso_query_begin(None)
             self.assertEqual(resp.status, 302)
-            self.assertEqual(resp.headers['Location'], (
-                "https://example.os.com:5001/v3" +
-                _path +
-                "?origin={origin}".format(
-                    origin="https://localhost/login/websso"
-                )
-            ))
+            self.assertEqual(
+                resp.headers["Location"],
+                (
+                    "https://example.os.com:5001/v3"
+                    + _path
+                    + "?origin={origin}".format(origin="https://localhost/login/websso")
```

Comment:
```suggestion
                    + "?origin=https://localhost/login/websso"
```
we don't really need a formatting here